expression/builtin_info_test.go

```diff
@@ -151,10 +151,16 @@ func (s *testEvaluatorSuite) TestCollation(c *C) {
 
 func (s *testEvaluatorSuite) TestRowCount(c *C) {
 	defer testleak.AfterTest(c)()
+	ctx := mock.NewContext()
+	sessionVars := ctx.GetSessionVars()
+	sessionVars.PrevAffectedRows = 10
+
 	fc := funcs[ast.RowCount]
-	f, err := fc.getFunction(s.ctx, s.datumsToConstants(types.MakeDatums()))
-	c.Assert(f, IsNil)
-	c.Assert(err, ErrorMatches, "*FUNCTION ROW_COUNT does not exist")
+	f, err := fc.getFunction(ctx, nil)
+	c.Assert(err, IsNil)
+	d, err := evalBuiltinFunc(f, nil)
```

Comment:
@mccxj We'd better not to use `evalBuiltinFunc()` in unit test.

Reply:
You can take a look at `TestArithmeticPlus()` at **expression/builtin_arithmetic_test.go** and rewrite this unit test like that.

Reply:
@zz-jason ok. PTAL.